src/app/utils/pinecone.ts

```diff
@@ -21,25 +20,34 @@ const getMatchesFromEmbeddings = async (embeddings: number[], topK: number, name
   // Retrieve the list of indexes
   const indexes = await pinecone.listIndexes()
 
+  let exists = false
```

Comment:
Another piece of feedback Github won't let me put in the right place: return type should be `Promise<ScoredPineconeRecord[]>`. The `ScoredVector` type export is still available from the module but we're trying to standardize on the terminology of "record" being the thing that is stored and fetched/queried from indexes. Relevant [docs](https://docs.pinecone.io/docs/overview#pinecone-indexes-store-records-with-vector-data).

Reply:
Makes sense, thanks for the explanation!

Reply:
Done in 14650f5